docs/UserGuide.md

```diff
@@ -1476,6 +1741,14 @@ Command Words Accepted: `exit`, `quit`, `bye` (case-insensitive)
 | **Clear Budget**            | `clear_budget`                                                                                        |
 | **Get Budget**              | `get_budget`                                                                                          |
 
+| **Reset UniCa$h to default** | `reset_unicash` (Add this to table)
+
+| **Show Help Window** | `help` (Add this to table)
+
+| **Show Welcome Message with Summary** | `help` (Add this to table)
+
+| **Show Command Specific Help** | `help [COMMAND_WORD]` (Add this to table)
+
```

Comment:
FYI, the table rows must be all together with no newlines in between, otherwise, it looks like this:

![Screenshot 2023-11-13 at 10 43 01](https://github.com/AY2324S1-CS2103-T16-3/tp/assets/29497029/6ed3dc88-d28c-485a-b036-b6df6578dfad)

Reply:
Actually for this part I intentionally left it this way so you can add it on your side, when i tried to use Intellij to auto-format, it overwrote the whole table, and I didn't want to affect the authorship unless you're ok w that then I can just do it on my side

Reply:
Ah understandable, will update it in my PR #211 then, thanks for noticing!

Reply:
Believe it's best to remove this from your side then since I've just added them in

Reply:
Thanks, will do.